src/dialogue_helpers.h

```diff
@@ -50,20 +50,19 @@ struct talk_effect_fun_t {
         void set_remove_active_mission( const JsonObject &jo, const std::string &member );
         void set_offer_mission( const JsonObject &jo, const std::string &member );
         void set_make_sound( const JsonObject &jo, const std::string &member, bool is_npc );
-        void set_run_eocs( const JsonObject &jo, std::string_view member );
-        void set_run_eoc_with( const JsonObject &jo, std::string_view member );
-        void set_run_eoc_until( const JsonObject &jo, std::string_view member );
+        void set_run_eocs( const JsonObject &jo, const std::string &member );
```

Comment:
I'd rather see the other ones moved to `std::string_view` than going backwards on this. Don't worry about churn as it's fully justified in this case.